discovery-provider/src/__init__.py

```diff
@@ -345,6 +346,18 @@ def configure_celery(flask_app, celery, test_config=None):
             "cache_trending_playlists": {
                 "task": "cache_trending_playlists",
                 "schedule": timedelta(minutes=30)
+            },
+            "update_aggregate_user": {
+                "task": "update_aggregate_user",
+                "schedule": timedelta(seconds=30)
```

Comment:
do we maybe want to offset these a bit? so everything isn't happening at once - not sure if real issue

Reply:
i dont think you can have sub-minute crons, so they have to be expressed this way with timedelta or by extending the scheduler interface https://docs.celeryproject.org/en/stable/reference/celery.schedules.html#celery.schedules.schedule which seems like a lot of complexity. 

I think we should probably pull this time delta down from 30 to somewhere around 15s if we can afford it, but spacing them out would be nice. any ideas on an easy way to do it? i think this is probably ok as is though

Reply:
I think we should leave it as it for now, and test more in the unregistered prod discovery box and look at the numbers from there if we want to change it before deploying